src/containers/note/toolbar.tsx

```diff
@@ -92,19 +118,71 @@ function Toolbar({ editor }: ToolbarProps) {
       ariaLabel: "H5",
       isActive: editor.isActive("heading", { level: 5 }),
     },
+    // 링크 버튼
+    {
+      onClick: openLinkModal,
+      IconComponent: LinkIcon,
+      ariaLabel: "Link",
+      isActive: false,
+    },
   ];
 
+  const linkInput = (
+    <input
+      type="text"
+      value={inputLink}
+      onChange={handleInputChange}
+      placeholder="https://"
+      className="h-44 w-full truncate rounded-12 bg-slate-50 p-5 text-sm md:h-48 md:text-base"
+    />
+  );
```

Comment:
input에 포커스되었을 때 테두리 스타일링이 없으면 좋을 것 같아요! 패딩은 피그마에서와 같이 수정하면 좋을 것 같습니다

```suggestion
  const linkInput = (
    <input
      type="text"
      value={inputLink}
      onChange={handleInputChange}
      placeholder="https://"
      className="h-44 w-full truncate rounded-12 bg-slate-50 px-24 py-12 text-sm outline-none md:h-48 md:text-base"
    />
  );
```